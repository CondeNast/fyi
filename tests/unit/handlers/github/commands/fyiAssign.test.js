const fyiAssignEvent = require('../../../fixtures/fyi-assign')
const models = require('../../../../../src/models')

let app
let github

describe('Arch Bot', () => {
  beforeEach(() => {
    ({app, github} = require('../../../setup.js'))
  })

  describe('FYI Assign', () => {
    it('Assigns Issue to Assignee in Repo Repository', async () => {
      await app.receive(fyiAssignEvent)
      expect(github.issues.addAssigneesToIssue).toMatchSnapshot()
    })
  })

  afterAll(async () => {
    await models.sequelize.close()
  })
})
