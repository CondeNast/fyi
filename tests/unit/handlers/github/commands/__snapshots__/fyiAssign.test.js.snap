// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arch Bot FYI Assign Assigns Issue to Assignee in Repo Repository 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      Object {
        "assignees": Array [
          "johnkpaul",
        ],
        "number": 1,
        "owner": "choosenearme",
        "repo": "stunning-journey",
      },
    ],
  ],
}
`;
